src/main/java/com/devshop/devshop/model/Order.java

```diff
@@ -0,0 +1,56 @@
+package com.devshop.devshop.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class Order {
+
+	@Id
+	@GeneratedValue(generator = "orderSeq")
+	@SequenceGenerator(name = "orderSeq", sequenceName = "order_seq", allocationSize = 1)
+	private long id;
+
+	@ManyToOne(targetEntity = User.class)
+	private User userId;
```

Comment:
Zmień proszę na user. Jeśli zostawisz tak jak jest to w bazie danych utworzy się kolumna o nazwie user_id_id.

Reply:
poprawione